sycl/include/sycl/builtins.hpp

```diff
@@ -48,6 +48,287 @@ template <class T, int N> marray<T, N> to_marray(vec<T, N> x) {
 namespace __sycl_std = __host_std;
 #endif
 
+#define __SYCL_DEF_BUILTIN_VEC(TYPE)                                           \
```

Comment:
The major part of these defines will be added with https://github.com/intel/llvm/pull/9753, here we have them just to have compileable tests.